internal/config/config_service.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/launchdarkly/ldcli/internal/resources"
+)
+
+type Service struct {
+	client resources.Client
+}
+
+func NewService(client resources.Client) Service {
+	return Service{
+		client: client,
+	}
+}
+
+// VerifyAccessToken is true if the given access token is valid to make API requests.
+func (s Service) VerifyAccessToken(accessToken string, baseURI string) bool {
```

Comment:
I could move more of the `config --set` code into the config domain if it would make the cmd more readable.